src/components/toolbar/legg-til-arbeidsliste-knapp.tsx

```diff
@@ -51,6 +52,7 @@ function ArbeidslisteKnapp() {
     return (
         <>
             <Button
+                style={{background: 'red'}}
```

Comment:
Vil bare verifisere at denne skal være rød? (eller var det bare for å sjekke noe?) om den skal være rød kan du jo vurdere å legge det til i en css fil isteden for å bruke style 👍